src/mame/nintendo/nes_vt32.cpp

```diff
@@ -468,6 +480,9 @@ CONS( 201?, mymman,    0,  0,  nes_vt32_8mb, nes_vt32, nes_vt32_unk_state, empty
 // most games work, a few minor graphical issues common to the same games in other units
 CONS( 202?, typo240,   0,  0,  nes_vt32_16mb, nes_vt32, nes_vt32_unk_state, empty_init, "Typo", "Vintage Gamer 240-in-1", MACHINE_IMPERFECT_GRAPHICS )
 
+// speed challenge doesn't work (mapper emulation? or wrong driver?)
```

Comment:
I've found it possible to get into Speed Challenge, albeit not very reliably. When it does hang, it (repeatedly) loops until the PPU Sprite 0 Hit flag becomes 1, which somehow fails to happen.

This one probably really is VT32 based; it even uses the $A1 XOR encryption.